.github/actions/e2e-arc-test/action.yaml

```diff
@@ -0,0 +1,44 @@
+name: 'E2E ARC Test Action'
+description: 'Includes common arc installation, setup and test file run'
+
+inputs:
+  github-token:
+    description: 'A Github PAT'
```

Comment:
Are we only accepting GitHub Personal Access Tokens here? What if we use a token from a GitHub App?

Reply:
No, it was an outdated description, thanks for catching this!